### DISCUSSION TITLE
condensed vs uncondensed

### DISCUSSION THREAD

**Discussion by RayanFarhat** [JWock82/Pynite#185] [Q&A]
Hey, I am wondering if there is any disadvantages to use the functions k_unc and fer_unc over k() and fer() in Member3D?
I mean other than make the analysis more simple.

**Comment by JWock82** [comment]
`k_unc` and `fer_unc` do not account for member end releases. End releases are applied using a process known as "static condensation". `unc` stands for "uncondensed".

**Comment by RayanFarhat** [comment]
Oh I see, thanks mate.
I do not use End releases, that's why it work on k_unc and fer_unc.
